app/controllers/games_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
     @sort_dir = params[:sort_dir] || Game.default_direction
 
     @games = Game.all.sort_table(@sort, @sort_dir)
+    @pagy, @games = pagy(@games, items: 20)
```

Comment:
`20` here is called a magic number. You don't know where it comes from, and have to "investigate" to understand what it means. I suggest adding maybe to application controller something like
```
PER_PAGE = 20
```
and then use it here